package.json

```
@@ -12,8 +12,9 @@
     "node": ">=16.18.0"
   },
   "scripts": {
-    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build'",
+    "start": "yarn typechain && nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn dev'",
     "build": "yarn run clean && yarn typechain && yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
+    "dev": "yarn run clean && yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
```

wdyt about dropping the `clean` target here? It _should_ be redundant in almost all scenarios I expect.

```suggestion
    "dev": "yarn run build:cjs & yarn run build:esm & yarn run build:types; wait",
```

---

Yeah makes sense!